package.json

```diff
@@ -153,7 +153,7 @@
     "handlebars": "^4.5.3",
     "q": "1.4.1",
     "u2f-api-polyfill": "0.4.3",
-    "underscore": "1.8.3"
+    "underscore": "1.13.1"
```

Comment:
This change is already merged https://github.com/okta/okta-signin-widget/pull/2007/files. Can you revert this on your PR and base your changes off the `5.7` branch and rebase ?

Reply:
The branch was already rebased on 5.7, but base branch in GitHub was master. Fixed